src/app/story/storyrow/storyrow.component.css

```diff
@@ -50,11 +42,77 @@
     margin: 0;
 }
 
+.node-display-name {
```

Comment:
Could you please use the BEM conventions for your CSS classes like the rest of the project does?
https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/